test/Bridges/lazy_bridge_optimizer.jl

```diff
@@ -1751,13 +1750,13 @@ function test_bridge_selection()
         bridged_mock,
         MOI.VectorOfVariables,
         MOI.RotatedSecondOrderCone,
-    ) == MOI.Bridges.Constraint.RSOCtoPSDBridge{
+    ) == MOI.Bridges.Constraint.RSOCtoSOCBridge{
```

Comment:
@blegat this test should have shown us the problem! It bridged RSOC to PSD, even though RSOC to SOC was available. It used to pick toPSD because that was the first bridge that was added.